js/ui/payg.js

```diff
@@ -457,6 +466,53 @@ function timeToString(seconds) {
     return ("%s %s").format(daysStr, hoursStr);
 }
 
+// Similar to timeToStrings, but does not process partial time,
+// since it's meant to be used for expiration time changes, with
+// a wider range of periods
+function successMessage() {
```

Comment:
here (and in a number of other places), please instead use the format `fooBar(args) { ... }` instead of `fooBar: function(args) {` or `function fooBar(args) {`

Reply:
I was retesting this and it seems that outside of class declarations the <function> argument is necessary so the file can identify the method, so I have left as it is.